tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -693,6 +692,8 @@ EOF
 		tar xvJpf ${AGENT_TARBALL} -C ${ROOTFS_DIR}
 	fi
 
+	${stripping_tool} ${ROOTFS_DIR}/usr/bin/kata-agent
```

Comment:
We can do this later, but perhaps we should have a way to bypass this new behavior.

Reply:
I'd prefer doing this later, but just to confirm.

The idea here is that we can bypass the behaviour and actually **not** have the component installed, correct?
If so, I think we can enforce this for the attestation-agent / confiential-data-hub / opa, are those are extra components.  The kata agent itself must always be there, though.

Reply:
Never mind. Yesterday I thought this rootfs.sh change was a significant change in symbol strip behavior - but I see now that it's consistent with the previous behavior.